andela/gimli-cfh
# #145026969 change task runner to gulp

#### What does this PR do?
This PR convert the task runner from grunt to gulp

#### Description of Task to be completed?
* Convert grunt to gulp
* Ensure all the gulp dependencies needed for the application is included
* Configure gulp to run mundane tasks

#### How should this be manually tested?
The app should run and listen to port 3000 when `npm start` is run in the command line

#### Any background context you want to provide?
* N/A

#### What are the relevant pivotal tracker stories?
*  Convert from Grunt to Gulp taskrunner

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
